src/utils/local.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isDevEnvironment = () =>
```

Comment:
I would rename this is `isLocalEnvironment` because `dev.easi.cms.gov` is the `dev` environment.